cumulus_library_data_metrics/base.py

```diff
@@ -75,6 +75,14 @@ def prepare_queries(
         self.extra_schema_checks(config)
         self.add_metric_queries()
 
+    def get_output_mode(self) -> str:
+        # TODO: add the ability for cumulus-library to take study args like
+        #  --study-option=output-mode:cube (or whatever)
```

Comment:
i know this is on our short list but for posterity: https://github.com/smart-on-fhir/cumulus-library/issues/247